common/config/webpack/sampleapp.webpack.config.js

```diff
@@ -77,6 +77,7 @@ const webpackConfig = (sampleAppDir, env, babelConfig) => {
         patterns: [
           { from: path.resolve(sampleAppDir, "public/manifest.json"), to: "manifest.json" },
           { from: path.resolve(sampleAppDir, "public/backgrounds"), to: "backgrounds",  noErrorOnMissing: true },
+          { from: path.resolve(sampleAppDir, "public/sounds"), to: "sounds",  noErrorOnMissing: true }
```

Comment:
to avoid needing this in future, we should have a public/assets folder and have that be copied over (then it has subfolders for backgrounds, sounds etc. but any new subfolder would be auto copied)

Reply:
(but thats longer than a one line change 😄)

Reply:
on it!